### ISSUE TITLE
General ROOMS cards

### ISSUE THREAD

**Issue by R0ck3tF** [Anashost/MY-HA-DASH#1]
I don't know why when I copy the card code I get this error, I haven't been able to solve it yet.
![Captura de pantalla 2023-08-18 105122](https://github.com/Anashost/MY-HA-DASH/assets/127993649/9cf3b4b5-0ae8-4233-b544-ef608fd2455b)

**Comment by Anashost** [comment]
Have you created the `sensor.salon_lights_of_count` & `group.salon_lights`, if yes double check it's state in: Developer tools > states

**Comment by R0ck3tF** [comment]
I am unable to color the sections "FLOOR, MEDIA, CLEAN etc." as it is in the YAML you have shared. What is falling me?

![image](https://github.com/Anashost/MY-HA-DASH/assets/127993649/5f3c9f95-6442-4cc0-bb61-234196432a55)

**Comment by Anashost** [comment]
Have u created the `input_boolean.on_state` like in step 4?

Drop me your yaml if problem is not fixed.

**Comment by R0ck3tF** [comment]
Yes, it has to be always "ON"? maybe that was the problem?

![image](https://github.com/Anashost/MY-HA-DASH/assets/127993649/b2e07fb6-a03d-4fd9-aadd-814dafbe285f)

**Comment by Anashost** [comment]
Yes this has to be always `on` as it's used for navigation buttons like mushroom-entity-card and mushroom-templete-card that needs an active entity to show colors.

**Comment by R0ck3tF** [comment]
ok ok, sorry to bother you, it was my problem for not knowing! your work has been incredible!

I am going to try to make the pop-up cards now.

**Comment by Anashost** [comment]
@R0ck3tF It's ok, I'm always happy to help so feel free

**Comment by ProjectGoose24** [comment]
> Have you created the `sensor.salon_lights_of_count` & `group.salon_lights`, if yes double check it's state in: Developer tools > states

In where do you create this sensor?

**Comment by Anashost** [comment]
@ProjectGoose24 it depends,
If you usually put your sensors in configurations.yaml directly then create it there.

If not! then it's recommended to do the following:

* In your config directory `config/` create  this file: `sensors.yaml` and add the sensors you want it it.
* then add this line in your configurations.yaml:
`sensor: !include sensors.yaml`

**Comment by Yunlliang** [comment]
Hello! How to fix this error
Error loading /config/configuration.yaml:   in "/config/configuration.yaml", line 17, column 12: Unable to read file /config/proximity.yaml.

**Comment by Anashost** [comment]
@Yunlliang

Cheak [step 6](https://github.com/Anashost/MY-HA-DASH?tab=readme-ov-file#6-you-need-to-create-these-additional-sensors) again, you need to create proximity.yaml and restart HA.
